torchtext/experimental/models/xlmr_model.py

```diff
@@ -0,0 +1,126 @@
+import torch
+import torch.nn as nn
+from torchtext.experimental.modules import TransformerEncoder
+from .xlmr_transform import XLMRTransform
+from torchtext.utils import download_from_url, extract_archive, load_args_from_json
+from torchtext.experimental.transforms import sentencepiece_tokenizer
+from torchtext.experimental.vocab import load_vocab_from_file
+
+
+# [TODO] Add torch.hub support
+# [TODO] Download file from manifold
+# [TODO] check base model config
+def xlmr_base(root='./.model'):
```

Comment:
I've commented on this already, why pass a `root` directory, and not instead rely on `torch.hub` caching strategy (which can be overriden by the user)?